src/content/docs/dropins/payment-services/containers/apple-pay.mdx

```diff
@@ -16,28 +16,58 @@ The `ApplePay` container provides the following configuration options:
   compact
   options={[
     ['Option', 'Type', 'Req?', 'Description'],
-    ['getCartId', 'function', 'Yes', 'Should return a promise that resolves to the shopper`s cart ID.'],
-    ['isVirtualCart', 'boolean', 'Yes', 'Whether the cart contains only virtual/downloadable products.'],
+    ['location', 'string', 'Yes', 'Location where the Apple Pay button is to be rendered. Must be either "CHECKOUT" or "PRODUCT_DETAIL".'],
+    ['getCartId', 'function', 'Maybe', 'Required if createCart not provided. Should return a promise that resolves to the shopper\'s cart ID.'],
```

Comment:
```suggestion
    ['getCartId', 'function', 'Maybe', 'Required if "createCart" not provided. Should return a promise that resolves to the shopper\'s cart ID.'],
```
for consistency with `onButtonClick`, which also references "code" in double quotes.